InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -2284,7 +2284,7 @@ private string ConditionalTextureNameOrUUID(SceneObjectPart part, UUID assetID)
 
             // We need to provide the name if present in the prim.
             string result = InventoryName(assetID);
-            if (result != "")
+            if (!String.IsNullOrEmpty(result))
                 return result;
```

Comment:
Possible semantic change: if some some reason InventoryName could return a null, not sure that's possible, returning a null here is still probably not a good idea.